libraries/api-documenter/src/yaml/YamlGenerator.ts

```diff
@@ -171,8 +268,25 @@ export class YamlGenerator {
     return yamlItem as IYamlItem;
   }
 
+  private _populateYamlStructure(yamlItem: Partial<IYamlItem>, docItem: DocItem): void {
```

Comment:
>Structure [](start = 23, length = 9)

Call this something other than "Structure" - "structure" implies the structure of the actual YAML file.